README.md

```diff
@@ -451,17 +451,7 @@ Once the nodes are up, you may now expose various services (using `k9s` (shift-f
   kubectl port-forward svc/haproxy-node3-svc -n "${SOLO_NAMESPACE}" 52211:50211 &
   ```
 * Envoy Proxy: `envoy-proxy-<node ID>-svc`
-  ```bash
-  # enable portforwarding for envoy proxy
-  kubectl port-forward svc/envoy-proxy-node1-svc -n "${SOLO_NAMESPACE}" 8181:8080 &
-  kubectl port-forward svc/envoy-proxy-node2-svc -n "${SOLO_NAMESPACE}" 8281:8080 &
-  kubectl port-forward svc/envoy-proxy-node3-svc -n "${SOLO_NAMESPACE}" 8381:8080 &
-  ```
-* Hedera explorer: `fullstack-deployment-hedera-explorer`
-  ```bash
-  #enable portforwarding for hedera explorer, can be access at http://localhost:8080/
-  kubectl port-forward svc/fullstack-deployment-hedera-explorer -n "${SOLO_NAMESPACE}" 8080:80 &
-  ```
+* Hedera explorer: `solo-deployment-hedera-explorer`
```

Comment:
you removed the bash examples for envoy-proxy-<node ID>-svc (which should be nodeAlias), and you removed the bash example for the hedera explorer, also.  Was this a an oversight?